File: msgraph-mail/go/utilities/users/mailFolders/messageRules/message_rules_response.go
package messagerules
import (
    "github.com/microsoft/kiota-samples/msgraph-mail/go/utilities/users"
    "github.com/microsoft/kiota/abstractions/go/serialization"
)
type MessageRulesResponse struct {
    additionalData Map<String, Object>;
    nextLink *string;
    value []MessageRule;
}

File: msgraph-mail/swiftconsoleapp/Sources/swiftconsoleapp/Users/Item/MailFolders/Item/MessageRules/MessageRulesResponse.swift
extension Swiftconsoleapp.Users.Item.MailFolders.Item.MessageRules {
    public class MessageRulesResponse : IAdditionalDataHolder {
        public var additionalData: IDictionary<string, object>
        public var nextLink: String
        public var value: [MessageRule]
    }
}
